### PR TITLE
feat (doc) more docs for bandits + improved tracker

### PR DESCRIPTION
**Pull request by sauxpa**

### CODE REVIEW

**Comment by sauxpa** [comment]
I have merged your doc with mine, lmk what you think

File: rlberry/agents/bandits/tools/tracker.py [hunk 1]
```diff
@@ -60,13 +106,32 @@ def reset_tracker(self):
...
+        return self.read_last_tag_value("total_reward", arm)
+
+    def mu_hat(self, arm):
+        return self.read_last_tag_value("mu_hat", arm)
+
+    def iw_total_reward(self, arm):
+        return self.read_last_tag_value("iw_total_reward", arm)
+
     def update(self, arm, reward, params={}):
-        self.add_scalar("t", self.read_last_tag_value("t") + 1)
```
Comment (TimotheeMathieu) [inline rlberry/agents/bandits/tools/tracker.py:64]: Maybe a one-line docstring for each of those methods would be good ? I mean, it is mostly obvious from the name of the reward but a little docstring would not hurt. In particular to explain what iw_total_reward is and what does the update function do.

**Review by TimotheeMathieu — COMMENTED** [review]
LGTM, I suggested some additional docstring, otherwise the new docstring are clear. 
Maybe change the name of the PR to also include "improvement upon tracker" also ?

**Review by TimotheeMathieu — APPROVED** [review]
Great, Ok for me. Thanks.

**Pull request merged**